### ISSUE TITLE
extra-files not updated with release type simple

### ISSUE THREAD

**Issue by danielo515** [google-github-actions/release-please-action#763]
### TL;DR

Thank you for this action! I’m trying to update some files with the generated version, but I’m having difficulty using the ﻿release-type simple with ﻿extra-files. I added some inline annotations to the places where I want the version to be updated, but they are being ignored.
For example, in my init.lua file I have:

```
obj.version = "1.0.0" -- x-release-please-version
```

Is there a way to resolve this issue?

### Expected behavior

I want the file init.lua to get the version updated in the place where I added the inline annotation

### Observed behavior

The action doesn't even take into account the file, doesn't log anything about it

### Action YAML

```yaml
- uses: google-github-actions/release-please-action@v3
      id: release
      with:
        release-type: simple
        package-name: StreamDeckButton.spoon
        extra-files: |
            init.lua
        token: ${{ secrets.GITHUB_TOKEN }}
```

### Log output

```text
2023-04-24T13:25:41.5322158Z Requested labels: ubuntu-latest
2023-04-24T13:25:41.5322527Z Job defined at: danielo515/StreamDeckButton.spoon/.github/workflows/build.yml@refs/heads/main
2023-04-24T13:25:41.5322683Z Waiting for a runner to pick up this job...
2023-04-24T13:25:41.7450962Z Job is waiting for a hosted runner to come online.
2023-04-24T13:25:45.3677590Z Job is about to start running on the hosted runner: GitHub Actions 2 (hosted)
2023-04-24T13:25:48.2138177Z Current runner version: '2.303.0'
2023-04-24T13:25:48.2177832Z ##[group]Operating System
2023-04-24T13:25:48.2178588Z Ubuntu
2023-04-24T13:25:48.2179077Z 22.04.2
2023-04-24T13:25:48.2179371Z LTS
2023-04-24T13:25:48.2180076Z ##[endgroup]
2023-04-24T13:25:48.2180666Z ##[group]Runner Image
2023-04-24T13:25:48.2181164Z Image: ubuntu-22.04
2023-04-24T13:25:48.2181526Z Version: 20230417.1
2023-04-24T13:25:48.2182539Z Included Software: https://github.com/actions/runner-images/blob/ubuntu22/20230417.1/images/linux/Ubuntu2204-Readme.md
2023-04-24T13:25:48.2183852Z Image Release: https://github.com/actions/runner-images/releases/tag/ubuntu22%2F20230417.1
2023-04-24T13:25:48.2184555Z ##[endgroup]
2023-04-24T13:25:48.2185194Z ##[group]Runner Image Provisioner
2023-04-24T13:25:48.2185616Z 2.0.139.1
2023-04-24T13:25:48.2185929Z ##[endgroup]
2023-04-24T13:25:48.2189480Z ##[group]GITHUB_TOKEN Permissions
2023-04-24T13:25:48.2190386Z Contents: write
2023-04-24T13:25:48.2190814Z Metadata: read
2023-04-24T13:25:48.2191484Z PullRequests: write
2023-04-24T13:25:48.2192037Z ##[endgroup]
2023-04-24T13:25:48.2197224Z Secret source: Actions
2023-04-24T13:25:48.2198202Z Prepare workflow directory
2023-04-24T13:25:48.3182739Z Prepare all required actions
2023-04-24T13:25:48.3431919Z Getting action download info
2023-04-24T13:25:48.6784844Z Download action repository 'actions/checkout@v3' (SHA:8e5e7e5ab8b370d6c329ec480221332ada57f0ab)
2023-04-24T13:25:49.3809042Z Download action repository 'google-github-actions/release-please-action@v3' (SHA:c078ea33917ab8cfa5300e48f4b7e6b16606aede)
2023-04-24T13:25:49.9287399Z Download action repository 'actions/upload-artifact@v2' (SHA:82c141cc518b40d92cc801eee768e7aafc9c2fa2)
2023-04-24T13:25:50.5274868Z Complete job name: build
2023-04-24T13:25:50.6570966Z ##[group]Run actions/checkout@v3
2023-04-24T13:25:50.6571533Z with:
2023-04-24T13:25:50.6571975Z   repository: danielo515/StreamDeckButton.spoon
2023-04-24T13:25:50.6572689Z   token: ***
2023-04-24T13:25:50.6573202Z   ssh-strict: true
2023-04-24T13:25:50.6573603Z   persist-credentials: true
2023-04-24T13:25:50.6573974Z   clean: true
2023-04-24T13:25:50.6574240Z   fetch-depth: 1
2023-04-24T13:25:50.6574612Z   lfs: false
2023-04-24T13:25:50.6574984Z   submodules: false
2023-04-24T13:25:50.6575284Z   set-safe-directory: true
2023-04-24T13:25:50.6575658Z ##[endgroup]
2023-04-24T13:25:51.0223412Z Syncing repository: danielo515/StreamDeckButton.spoon
2023-04-24T13:25:51.0225675Z ##[group]Getting Git version info
2023-04-24T13:25:51.0226424Z Working directory is '/home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon'
2023-04-24T13:25:51.0227279Z [command]/usr/bin/git version
2023-04-24T13:25:51.0227547Z git version 2.40.0
2023-04-24T13:25:51.0228694Z ##[endgroup]
2023-04-24T13:25:51.0249793Z Temporarily overriding HOME='/home/runner/work/_temp/3aab7950-58bf-48c1-a6e3-46d41ef7b657' before making global git config changes
2023-04-24T13:25:51.0250371Z Adding repository directory to the temporary git global config as a safe directory
2023-04-24T13:25:51.0251089Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon
2023-04-24T13:25:51.0252243Z Deleting the contents of '/home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon'
2023-04-24T13:25:51.0253103Z ##[group]Initializing the repository
2023-04-24T13:25:51.0253523Z [command]/usr/bin/git init /home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon
2023-04-24T13:25:51.0359845Z hint: Using 'master' as the name for the initial branch. This default branch name
2023-04-24T13:25:51.0360418Z hint: is subject to change. To configure the initial branch name to use in all
2023-04-24T13:25:51.0360940Z hint: of your new repositories, which will suppress this warning, call:
2023-04-24T13:25:51.0361268Z hint: 
2023-04-24T13:25:51.0361654Z hint: 	git config --global init.defaultBranch <name>
2023-04-24T13:25:51.0361950Z hint: 
2023-04-24T13:25:51.0362363Z hint: Names commonly chosen instead of 'master' are 'main', 'trunk' and
2023-04-24T13:25:51.0363321Z hint: 'development'. The just-created branch can be renamed via this command:
2023-04-24T13:25:51.0363645Z hint: 
2023-04-24T13:25:51.0363939Z hint: 	git branch -m <name>
2023-04-24T13:25:51.0374869Z Initialized empty Git repository in /home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon/.git/
2023-04-24T13:25:51.0391873Z [command]/usr/bin/git remote add origin https://github.com/danielo515/StreamDeckButton.spoon
2023-04-24T13:25:51.0445239Z ##[endgroup]
2023-04-24T13:25:51.0446000Z ##[group]Disabling automatic garbage collection
2023-04-24T13:25:51.0449168Z [command]/usr/bin/git config --local gc.auto 0
2023-04-24T13:25:51.0495891Z ##[endgroup]
2023-04-24T13:25:51.0499395Z ##[group]Setting up auth
2023-04-24T13:25:51.0513828Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2023-04-24T13:25:51.0564216Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2023-04-24T13:25:51.1010231Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2023-04-24T13:25:51.1050414Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2023-04-24T13:25:51.1382854Z [command]/usr/bin/git config --local http.https://github.com/.extraheader AUTHORIZATION: basic ***
2023-04-24T13:25:51.1436203Z ##[endgroup]
2023-04-24T13:25:51.1437164Z ##[group]Fetching the repository
2023-04-24T13:25:51.1452541Z [command]/usr/bin/git -c protocol.version=2 fetch --no-tags --prune --progress --no-recurse-submodules --depth=1 origin +e6363456032da902fc4bed17c2185ef72efd4b5f:refs/remotes/origin/main
2023-04-24T13:25:51.5669858Z remote: Enumerating objects: 13, done.        
2023-04-24T13:25:51.5670246Z remote: Counting objects:   7% (1/13)        
2023-04-24T13:25:51.5670586Z remote: Counting objects:  15% (2/13)        
2023-04-24T13:25:51.5670889Z remote: Counting objects:  23% (3/13)        
2023-04-24T13:25:51.5671190Z remote: Counting objects:  30% (4/13)        
2023-04-24T13:25:51.5671473Z remote: Counting objects:  38% (5/13)        
2023-04-24T13:25:51.5671775Z remote: Counting objects:  46% (6/13)        
2023-04-24T13:25:51.5672072Z remote: Counting objects:  53% (7/13)        
2023-04-24T13:25:51.5672354Z remote: Counting objects:  61% (8/13)        
2023-04-24T13:25:51.5672651Z remote: Counting objects:  69% (9/13)        
2023-04-24T13:25:51.5672949Z remote: Counting objects:  76% (10/13)        
2023-04-24T13:25:51.5673266Z remote: Counting objects:  84% (11/13)        
2023-04-24T13:25:51.5673550Z remote: Counting objects:  92% (12/13)        
2023-04-24T13:25:51.5673955Z remote: Counting objects: 100% (13/13)        
2023-04-24T13:25:51.5674258Z remote: Counting objects: 100% (13/13), done.        
2023-04-24T13:25:51.5674559Z remote: Compressing objects:  11% (1/9)        
2023-04-24T13:25:51.5675135Z remote: Compressing objects:  22% (2/9)        
2023-04-24T13:25:51.5675443Z remote: Compressing objects:  33% (3/9)        
2023-04-24T13:25:51.5675730Z remote: Compressing objects:  44% (4/9)        
2023-04-24T13:25:51.5676029Z remote: Compressing objects:  55% (5/9)        
2023-04-24T13:25:51.5676326Z remote: Compressing objects:  66% (6/9)        
2023-04-24T13:25:51.5676608Z remote: Compressing objects:  77% (7/9)        
2023-04-24T13:25:51.5676908Z remote: Compressing objects:  88% (8/9)        
2023-04-24T13:25:51.5677201Z remote: Compressing objects: 100% (9/9)        
2023-04-24T13:25:51.5678013Z remote: Compressing objects: 100% (9/9), done.        
2023-04-24T13:25:51.5678767Z remote: Total 13 (delta 0), reused 6 (delta 0), pack-reused 0        
2023-04-24T13:25:51.5680023Z From https://github.com/danielo515/StreamDeckButton.spoon
2023-04-24T13:25:51.5680557Z  * [new ref]         e6363456032da902fc4bed17c2185ef72efd4b5f -> origin/main
2023-04-24T13:25:51.5713206Z ##[endgroup]
2023-04-24T13:25:51.5713846Z ##[group]Determining the checkout info
2023-04-24T13:25:51.5715441Z ##[endgroup]
2023-04-24T13:25:51.5715856Z ##[group]Checking out the ref
2023-04-24T13:25:51.5721754Z [command]/usr/bin/git checkout --progress --force -B main refs/remotes/origin/main
2023-04-24T13:25:51.5782210Z Switched to a new branch 'main'
2023-04-24T13:25:51.5784363Z branch 'main' set up to track 'origin/main'.
2023-04-24T13:25:51.5793708Z ##[endgroup]
2023-04-24T13:25:51.5838477Z [command]/usr/bin/git log -1 --format='%H'
2023-04-24T13:25:51.5872518Z 'e6363456032da902fc4bed17c2185ef72efd4b5f'
2023-04-24T13:25:51.6307621Z ##[group]Run google-github-actions/release-please-action@v3
2023-04-24T13:25:51.6308288Z with:
2023-04-24T13:25:51.6308501Z   release-type: simple
2023-04-24T13:25:51.6308766Z   package-name: StreamDeckButton.spoon
2023-04-24T13:25:51.6309016Z   extra-files: init.lua

2023-04-24T13:25:51.6309480Z   token: ***
2023-04-24T13:25:51.6309775Z   fork: false
2023-04-24T13:25:51.6309990Z   clean: true
2023-04-24T13:25:51.6310224Z   bump-minor-pre-major: false
2023-04-24T13:25:51.6310510Z   bump-patch-for-minor-pre-major: false
2023-04-24T13:25:51.6310797Z   changelog-host: https://github.com
2023-04-24T13:25:51.6311090Z   github-api-url: https://api.github.com
2023-04-24T13:25:51.6311418Z   github-graphql-url: https://api.github.com/graphql
2023-04-24T13:25:51.6311691Z   monorepo-tags: false
2023-04-24T13:25:51.6311910Z ##[endgroup]
2023-04-24T13:25:52.3624671Z ✔ Looking for latest release on branch: main with prefix: StreamDeckButton.spoon
2023-04-24T13:25:52.3626549Z ❯ Fetching merge commits on branch main with cursor: undefined
2023-04-24T13:25:53.3338322Z ❯ Found latest release pull request: 1 version: 1.0.0
2023-04-24T13:25:53.3343426Z ❯ Fetching releases with cursor undefined
2023-04-24T13:25:53.6464118Z ❯ found release for StreamDeckButton.spoon Version {
2023-04-24T13:25:53.6464918Z   major: 1,
2023-04-24T13:25:53.6465544Z   minor: 0,
2023-04-24T13:25:53.6466581Z   patch: 0,
2023-04-24T13:25:53.6466974Z   preRelease: undefined,
2023-04-24T13:25:53.6517324Z   build: undefined
2023-04-24T13:25:53.6517551Z }
2023-04-24T13:25:53.6518141Z ❯ found 2 possible releases. [
2023-04-24T13:25:53.6518551Z   Version {
2023-04-24T13:25:53.6518745Z     major: 1,
2023-04-24T13:25:53.6519112Z     minor: 0,
2023-04-24T13:25:53.6519457Z     patch: 0,
2023-04-24T13:25:53.6519723Z     preRelease: undefined,
2023-04-24T13:25:53.6519963Z     build: undefined
2023-04-24T13:25:53.6520177Z   },
2023-04-24T13:25:53.6520359Z   Version {
2023-04-24T13:25:53.6520571Z     major: 1,
2023-04-24T13:25:53.6520803Z     minor: 0,
2023-04-24T13:25:53.6520993Z     patch: 0,
2023-04-24T13:25:53.6521394Z     preRelease: undefined,
2023-04-24T13:25:53.6521643Z     build: undefined
2023-04-24T13:25:53.6521841Z   }
2023-04-24T13:25:53.6522033Z ]
2023-04-24T13:25:53.6522483Z ✔ Building releases
2023-04-24T13:25:53.6522942Z ✔ Building strategies by path
2023-04-24T13:25:53.6523374Z ❯ .: simple
2023-04-24T13:25:54.0470934Z ✔ Looking for latest release on branch: main with prefix: StreamDeckButton.spoon
2023-04-24T13:25:54.0471527Z ❯ Fetching merge commits on branch main with cursor: undefined
2023-04-24T13:25:54.7503543Z ❯ Found latest release pull request: 1 version: 1.0.0
2023-04-24T13:25:54.7504786Z ❯ Fetching releases with cursor undefined
2023-04-24T13:25:55.0301629Z ❯ found release for StreamDeckButton.spoon Version {
2023-04-24T13:25:55.0301996Z   major: 1,
2023-04-24T13:25:55.0302231Z   minor: 0,
2023-04-24T13:25:55.0302463Z   patch: 0,
2023-04-24T13:25:55.0302692Z   preRelease: undefined,
2023-04-24T13:25:55.0302999Z   build: undefined
2023-04-24T13:25:55.0303228Z }
2023-04-24T13:25:55.0303538Z ❯ found 2 possible releases. [
2023-04-24T13:25:55.0303802Z   Version {
2023-04-24T13:25:55.0304029Z     major: 1,
2023-04-24T13:25:55.0304239Z     minor: 0,
2023-04-24T13:25:55.0304463Z     patch: 0,
2023-04-24T13:25:55.0304714Z     preRelease: undefined,
2023-04-24T13:25:55.0304956Z     build: undefined
2023-04-24T13:25:55.0305737Z   },
2023-04-24T13:25:55.0305950Z   Version {
2023-04-24T13:25:55.0306160Z     major: 1,
2023-04-24T13:25:55.0306380Z     minor: 0,
2023-04-24T13:25:55.0306606Z     patch: 0,
2023-04-24T13:25:55.0306835Z     preRelease: undefined,
2023-04-24T13:25:55.0307094Z     build: undefined
2023-04-24T13:25:55.0307319Z   }
2023-04-24T13:25:55.0307506Z ]
2023-04-24T13:25:55.0310621Z ✔ Building pull requests
2023-04-24T13:25:55.0311011Z ✔ Building strategies by path
2023-04-24T13:25:55.0320555Z ❯ .: simple
2023-04-24T13:25:55.0320901Z ✔ Collecting release commit SHAs
2023-04-24T13:25:55.0321244Z ❯ release search depth: 400
2023-04-24T13:25:55.0322455Z ❯ Fetching releases with cursor undefined
2023-04-24T13:25:55.3083872Z ❯ Found release for path ., v1.0.0
2023-04-24T13:25:55.3090238Z ❯ release for path: ., version: 1.0.0, sha: e0e7a47f2139ba1023cba2dadcd23e6b472d8318
2023-04-24T13:25:55.3091107Z ✔ Collecting commits since all latest releases
2023-04-24T13:25:55.3091645Z ❯ commit search depth: 500
2023-04-24T13:25:55.3096069Z ❯ Set(1) { 'e0e7a47f2139ba1023cba2dadcd23e6b472d8318' }
2023-04-24T13:25:55.3097120Z ❯ Fetching merge commits on branch main with cursor: undefined
2023-04-24T13:25:56.0018016Z ❯ Backfilling file list for commit: e6363456032da902fc4bed17c2185ef72efd4b5f
2023-04-24T13:25:56.3150921Z ❯ Found 1 files
2023-04-24T13:25:56.3151741Z ❯ Backfilling file list for commit: 23c62a072638eebc81f028c8b8f8d0510091439c
2023-04-24T13:25:56.5617319Z ❯ Found 2 files
2023-04-24T13:25:56.5618099Z ❯ Backfilling file list for commit: d8892f54ada06219454ee9a0047d3374daf7496f
2023-04-24T13:25:56.8289271Z ❯ Found 1 files
2023-04-24T13:25:56.8290187Z ❯ Backfilling file list for commit: 742728c9cf83989925e95fd696c6d906763a2114
2023-04-24T13:25:57.1025454Z ❯ Found 2 files
2023-04-24T13:25:57.1026723Z ❯ Backfilling file list for commit: 656e256b6b535ae4cb0f7812f20c752446896ead
2023-04-24T13:25:57.3590830Z ❯ Found 1 files
2023-04-24T13:25:57.3591862Z ❯ Backfilling file list for commit: 79b408fec3e36b00d77f4820d7d27c8a7a27a272
2023-04-24T13:25:57.5977573Z ❯ Found 1 files
2023-04-24T13:25:57.5978657Z ❯ Backfilling file list for commit: f904203fcccde3d8b2641b7281be7feb47619f14
2023-04-24T13:25:57.8493861Z ❯ Found 3 files
2023-04-24T13:25:57.8494411Z ❯ Backfilling file list for commit: e4feb2635b8c2f537551251da25a7085effbc513
2023-04-24T13:25:58.1049387Z ❯ Found 1 files
2023-04-24T13:25:58.1049930Z ❯ Backfilling file list for commit: acb0381d475cb18d732b8754dd96b6314436fe4b
2023-04-24T13:25:58.3665451Z ❯ Found 1 files
2023-04-24T13:25:58.3665958Z ❯ Backfilling file list for commit: 735a71d495abb9c55f1e4f08046d79ee5d0d2b9b
2023-04-24T13:25:58.6196592Z ❯ Found 1 files
2023-04-24T13:25:58.6197123Z ❯ Backfilling file list for commit: 49d38535d54469b453275e691468f6b3a96000ca
2023-04-24T13:25:58.8355561Z ❯ Found 1 files
2023-04-24T13:25:58.8356391Z ❯ Backfilling file list for commit: 0c9135ec273689e0e7a3768f377e65f6e6eeda9c
2023-04-24T13:25:59.0580882Z ❯ Found 1 files
2023-04-24T13:25:59.0581815Z ❯ Backfilling file list for commit: 5b05ddd0da24741a0ccbd092533431708a59f8a8
2023-04-24T13:25:59.3350116Z ❯ Found 2 files
2023-04-24T13:25:59.3350615Z ❯ Backfilling file list for commit: d964c4d98e503fc3d0c9028a97e49f3a001f4d83
2023-04-24T13:25:59.5726250Z ❯ Found 4 files
2023-04-24T13:25:59.5726957Z ❯ Backfilling file list for commit: 3dc39804abc720d99039e712fc08f472755b8c17
2023-04-24T13:25:59.8393904Z ❯ Found 2 files
2023-04-24T13:25:59.8394473Z ❯ Backfilling file list for commit: cf892d33ecbfebf84c536934c493f974b69568dd
2023-04-24T13:26:00.0687617Z ❯ Found 2 files
2023-04-24T13:26:00.0688732Z ❯ Backfilling file list for commit: d1b499117e8df7c7159c7b03e313a249c14909d1
2023-04-24T13:26:00.3329783Z ❯ Found 1 files
2023-04-24T13:26:00.3330925Z ❯ Backfilling file list for commit: 13d690f9e94ff1562f07544d62bb55b6a219fe63
2023-04-24T13:26:00.5698313Z ❯ Found 2 files
2023-04-24T13:26:00.5701459Z ✔ Splitting 4 commits by path
2023-04-24T13:26:00.5713753Z ✔ Building candidate release pull request for path: .
2023-04-24T13:26:00.5714122Z ❯ type: simple
2023-04-24T13:26:00.5716116Z ❯ targetBranch: main
2023-04-24T13:26:00.5757842Z ❯ commits: 4
2023-04-24T13:26:00.5760030Z ✔ Considering: 4 commits
2023-04-24T13:26:00.5764519Z ❯ component: 
2023-04-24T13:26:00.5765174Z ❯ pull request title pattern: undefined
2023-04-24T13:26:00.9993083Z ✔ Looking for open release pull requests
2023-04-24T13:26:01.3007521Z ✔ found 1 open release pull requests.
2023-04-24T13:26:01.3009229Z ✔ Looking for snoozed release pull requests
2023-04-24T13:26:01.6257616Z ✔ found 0 snoozed release pull requests.
2023-04-24T13:26:01.6264542Z ✔ PR https://github.com/danielo515/StreamDeckButton.spoon/pull/2 remained the same
2023-04-24T13:26:01.6563992Z Post job cleanup.
2023-04-24T13:26:01.7907559Z [command]/usr/bin/git version
2023-04-24T13:26:01.7968186Z git version 2.40.0
2023-04-24T13:26:01.8043220Z Temporarily overriding HOME='/home/runner/work/_temp/074f45b1-b6b4-432b-b1b9-2512d13d443f' before making global git config changes
2023-04-24T13:26:01.8049089Z Adding repository directory to the temporary git global config as a safe directory
2023-04-24T13:26:01.8060278Z [command]/usr/bin/git config --global --add safe.directory /home/runner/work/StreamDeckButton.spoon/StreamDeckButton.spoon
2023-04-24T13:26:01.8121612Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2023-04-24T13:26:01.8178599Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :"
2023-04-24T13:26:01.8479530Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2023-04-24T13:26:01.8503287Z http.https://github.com/.extraheader
2023-04-24T13:26:01.8523526Z [command]/usr/bin/git config --local --unset-all http.https://github.com/.extraheader
2023-04-24T13:26:01.8566192Z [command]/usr/bin/git submodule foreach --recursive sh -c "git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :"
2023-04-24T13:26:01.9130270Z Cleaning up orphan processes
```

### Additional information

https://github.com/danielo515/StreamDeckButton.spoon/pull/2

**Comment by danielo515** [comment]
Ok, after 3 release PRs it has started to work. I think the root cause of the problem was that the versions didn't match, and therefore it was silently rejecting to update it.
My fist attempt looked like:

```lua
-- x-release-please-start-version
obj.version = "1.0"
-- x-release-please-end
```

Then I tried inlining, and update the version to be 1.0.0, but by that time, the latest releaase was already 2.0.0 so again release please was rejecting to update it.
Was not until I made them match that it started working. Does that make sense?

**Comment by chingor13** [comment]
Apologies for the super late reply. 

The generic file updater looks for semver looking entries (needs all 3 major/minor/patch numbers).
